extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayRecorder.java

```diff
@@ -80,4 +92,21 @@ public void doStartActions() {
             }
         }
     }
+
+    static class BaselineCommand implements FlywayExecutor.Command<BaselineResult> {
+        BaselineCommand(Flyway flyway) {
+            this.flyway = flyway;
+        }
+
+        final Flyway flyway;
+
+        @Override
+        public BaselineResult execute(CompositeMigrationResolver cmr, SchemaHistory schemaHistory, Database d,
+                Schema defaultSchema, Schema[] s, CallbackExecutor ce, StatementInterceptor si) {
+            if (!schemaHistory.exists()) {
```

Comment:
You said previously:
> if the flyway_schema_history table exist and contains any element, everything is ok.

Why just checking if the `flyway_schema_history` does not exist here?